screens/Issuers/IssuersScreen.tsx

```diff
@@ -124,24 +129,56 @@ export const IssuersScreen: React.FC<
     );
   }
 
+  const showAllIssuers = () => {
+    setFilteredSearchData(issuers);
+  };
+
+  const searchIssuer = text => {
+    if (text === '') {
+      setFilteredSearchData(issuers);
+      setSearch(text);
+    } else {
+      let filterdData = issuers.filter(item => {
+        if (
+          getDisplayObjectForCurrentLanguage(item.display)
+            ?.name.toLowerCase()
+            .includes(text.toLowerCase())
+        ) {
+          return getDisplayObjectForCurrentLanguage(item.display);
+        }
+      }, []);
+
+      setFilteredSearchData(filterdData);
+      setSearch(text);
+    }
+  };
+
   return (
     <React.Fragment>
       {controller.issuers.length > 0 && (
-        <Column style={Theme.issuersScreenStyles.issuerListOuterContainer}>
```

Comment:
why are we removing this subHeader from the screen, is it not needed ?

Reply:
Currently, there is no wireframe for this so I removed based on the text inside of that subheader.
I think it was diff. while implementing a search bar. If required, I will revert it.